src/pseudopeople/interface.py

```diff
@@ -449,9 +457,10 @@ def generate_social_security(
         to a configuration YAML file, a configuration dictionary,
         or the sentinel value `pseudopeople.NO_NOISE`, which will generate a
         dataset without any configurable noise.
-    :param year: The latest year (format YYYY) to include in the dataset; will also
-        include all previous years. Will raise a ValueError if there is no data for
-        this year. If None is provided, data from all years are included in the dataset.
+    :param year: The final year of social security records to include in the dataset (format YYYY, e.g., 2036); will also
```

Comment:
```suggestion
    :param year: The final year of simulated social security records to include in the dataset (format YYYY, e.g., 2036); will also
```

You have this in some of the other docstrings, and this might be the _most_ important to emphasize is simulated 😄